INSTALL.md

```diff
@@ -163,6 +163,30 @@ grunt.initConfig({
 });
 ```
 
+## Rollup
+
+Complete [PostCSS CLI](#postcss-cli) setup.
+
+Add [Rollup Plugin PostCSS] to your project:
+
+```bash
+npm install rollup-plugin-postcss --save-dev
+```
+
+Use [Rollup Plugin PostCSS] in your rollup.config.js:
+
+```js
+import postcss from 'rollup-plugin-postcss';
+
+module.exports = {
+  input: '...',
+  output: {...},
+  plugins: [
+    postcss({/* options */ })
```

Comment:
To keep consistency with the other examples, I would add.

```
    postcss({
      plugins: [
        postcssPresetEnv({ /* pluginOptions */ })
      ]
    })
```

Reply:
i see what you mean! 

here's where my head is/was at:
- my first instruction of this rollup config flow is to [complete the postcss-cli setup](https://github.com/csstools/postcss-preset-env/pull/153/files#diff-7d442b7eb49f5fc377f51e74b291cfc1R168), which will have them create a `postcss.config`, which has them setup `postcss-preset-env`
- since this helper section was for rollup, i felt like it should be rollup config options only, and let setting up `postcss-preset-env` be a set of instructions elsewhere

if i use the suggestion from @nucliweb, i'd need to add more instruction right?
1. `npm install postcss-preset-env`
1. import `postcss-preset-env` in the rollup.config
1. pass `postcss-present-env` plugin to `postcss` which get's passed to rollup..

all in all, i def love where your heads at in the review! consistency ftw! i guess i opted to link to the instructions then duplicate them. thoughts? i can go either way, editing a markdown file is easy! getting the copy/instructions so their usable and helpful, much harder 🙂

Reply:
You’re right, @argyleink. In any project, I recommend adding it to package.json or through a config file. So, I suppose I should probably follow my own advice...

I’m thinking now. Great feedback! Thank you.

Reply:
We can update the documentation of intalls, ~I can help you~ it would be an honor to helpt you 😊

Reply:
Thanks for the offer. I will go ahead and merge Adam’s generous contribution. This work only helps, and it should make additional changes easier.

And then look at us developing open source iteratively. 🥇

Reply:
I hope this only helps you, if you end up having the time to contribute that, @nucliweb!